source/installation/index.md

```diff
@@ -1,14 +1,17 @@
 # Installation
 
-This section describes how to install NodeX. As we have explained, NodeX consists of NodeX Hub/Studio, a service for administrators, and NodeX AGENT, which is installed on edge devices. No special procedures are required to start using NodeX Hub/Studio, and a single email address is all you need to start your trial right away. NodeX AGENT, on the other hand, must be embedded in an edge device and is now available in executable binaries for the x86_64 architectures of the Linux OS. NodeX AGENT consists of a single binary with no dependencies and can be easily installed.
+This section describes how to install NodeX Agent. NodeX AGENT, must be embedded in an edge device and is now available in executable binaries for the x86_64 architectures of the Linux OS and the Mac OS. NodeX AGENT consists of a single binary with no dependencies and can be easily installed.
 
 ```{note}
-You can register to NodeX Hub/Studio at [https://studio.nodecross.io](https://studio.nodecross.io) (under development).
+NodeX has a service for administrators, NodeX Studio, which requires installation of the NodeX Agent and setup to start using NodeX Studio.
+To start using NodeX Studio, please contact Collabogate Japan.
```

Comment:
I think the company name should use CollaboGate Japan, Inc..
ref. https://nodecross.io/about
```suggestion
To start using NodeX Studio, please contact [CollaboGate Japan, Inc.](https://nodecross.io/#demo-request-form)
```